rulesets/faang_experiments.metadata_rules.json

```diff
@@ -324,53 +324,38 @@
       "rules": [
         {
           "mandatory": "mandatory",
-          "name": "target protein name",
-          "description": "The name of the antibody-targeted protein. For histone modification experiment, please use 'Histone' in this field and use the separate histone modification type field to describe the modification type, e.g. 'H3K27_trimethylation_site'",
+          "name": "chip target",
+          "description": "The particular target which the ChIP-seq is aiming for. If your target is not in the list, please contact faang-dcc to add them",
```

Comment:
"The target of the ChIP-seq experiment e.g. H3K4Me3, H3K4Me1, H3K27Me3, H3K27Ac, CTCF. If your target is not in the list, please contact the faang-dcc helpdesk."

Reply:
addressed in 6cf1f37